tests/create_katsu_datasets.sh

```diff
@@ -3,32 +3,32 @@ cd tests
 export PROJECT_ID=$(curl -X POST 'localhost:8001/api/projects' -H "Content-Type: application/json" -d '{"title": "tests", "description": "Project for tests"}'|jq -r .identifier)
 echo "Created project ${PROJECT_ID}";echo
 
-export OPEN_DATASET1_ID=$(cat dataset_template.json|sed s/DATASET_TITLE/open1/|sed s/PROJECT_ID/${PROJECT_ID}/|curl -X POST 'localhost:8001/api/datasets' \
+export OPEN_DATASET1_ID=$(cat templates/dataset_template.json|sed s/DATASET_TITLE/open1/|sed s/PROJECT_ID/${PROJECT_ID}/|curl -X POST 'localhost:8001/api/datasets' \
 -H 'Content-Type: application/json' \
```

Comment:
I'd suggest pulling out this repetitive cat/sed/sed/curl one-liner into a bash function